.github/workflows/cli.yml

```diff
@@ -40,26 +40,39 @@ jobs:
         ../packages/cli/dist/bin/index.js dataset --help
         ../packages/cli/dist/bin/index.js bip --help
         ../packages/cli/dist/bin/index.js serve --help
-    - name: test client basic operate init
+  basic-usage:
+    name: Test CLI on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install
+        npm run build
+    - name: create workspace
+      run: mkdir -p myworkspace
+    - name: test client basic usage init
       working-directory: ./myworkspace
       run: |
         mkdir init1
         cd init1
         ../../packages/cli/dist/bin/index.js init
+        ../../packages/cli/dist/bin/index.js run ./example/pipelines/text-bayes-classification.json
         cd ../
         mkdir init2
         cd init2
-        ../../packages/cli/dist/bin/index.js init --beta
-        cd ../
-        mkdir init3
-        cd init3
         ../../packages/cli/dist/bin/index.js init --tuna
```

Comment:
We don't need to test the `--tuna` on default GitHub Action, maybe we can test it on our custom server at China.

Reply:
removed.